### ISSUE TITLE
Prebuilt docker images are missing

### ISSUE THREAD

**Issue by TBX3D** [lifting-bits/rellic#339]
I've been struggling to build rellic, so I tried to install one of the docker images, but the page returns a 404 error. Please review.

**Comment by frabert** [comment]
I'm not in charge of the Docker images so right now I can't help with that, but could you elaborate on the issues you've been having building rellic?

**Comment by TBX3D** [comment]
I've kind of came to the conclusion that you can't build rellic or cxx-commons in debian, so I'm going to try it in ubuntu (I think that's what it is for the project?)

**Comment by frabert** [comment]
I believe it's more of a "it has never been tried" more than a "it can't be done", but your point stands.

In any case, I would probably be able to provide more guidance if you share which steps are failing, and how :)

**Comment by TBX3D** [comment]
that sounds good, I'll find some time I'm free and we could potentially figure it out.

**Comment by TBX3D** [comment]
hello, are you still interested in checking this out?

**Comment by frabert** [comment]
Yes, feel free to share any details on what you're encountering and I can try to help with that

**Comment by TBX3D** [comment]
an error occurs when building rellic in debian (WSL).
all dependencies seem to be up to date and were installed through the apt package manager


```
FAILED: lib/CMakeFiles/rellic.dir/AST/IRToASTVisitor.cpp.o
/usr/bin/c++ -DGFLAGS_IS_A_DLL=0 -I/home/owo/rellic/include -isystem /home/owo/lifting-bits-downloads/vcpkg_ubuntu-22.04_llvm-16_amd64/installed/x64-linux-rel/include -O2 -g -DNDEBUG -fvisibility=hidden -fvisibility-inlines-hidden -Werror -pedantic -Wno-unreachable-code-return -MD -MT lib/CMakeFiles/rellic.dir/AST/IRToASTVisitor.cpp.o -MF lib/CMakeFiles/rellic.dir/AST/IRToASTVisitor.cpp.o.d -o lib/CMakeFiles/rellic.dir/AST/IRToASTVisitor.cpp.o -c /home/owo/rellic/lib/AST/IRToASTVisitor.cpp
/home/owo/rellic/lib/AST/IRToASTVisitor.cpp: In member function ‘clang::Expr* rellic::ExprGen::visitCastInst(llvm::CastInst&)’:
/home/owo/rellic/lib/AST/IRToASTVisitor.cpp:899:7: error: attributes at the beginning of statement are ignored [-Werror=attributes]
  899 |       [[clang::fallthrough]];
      |       ^~~~~~~~~~~~~~~~~~~~~~
At global scope:
cc1plus: note: unrecognized command-line option ‘-Wno-unreachable-code-return’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
```

**Comment by frabert** [comment]
Can you post the output of `c++ --version`?

**Comment by pgoodman** [comment]
This could probably be changed to [`[[fallthrough]];`](https://en.cppreference.com/w/cpp/language/attributes/fallthrough) now.

**Comment by frabert** [comment]
True! @TBX3D can you try mass-substituting every `[[clang::fallthrough]];` to `[[fallthrough]];`, then recompiling?

**Comment by TBX3D** [comment]
I'd like if you could specify where to mass-substitute. I've been getting multiple syntax errors relating to fallthrough when substitutiung all of the vcpkg and rellic itself. I'm pretty sure I'm doing something wrong and would like some help

**Comment by TBX3D** [comment]
and also c++ --version:
```
$ c++ --version
c++ (Debian 13.3.0-3) 13.3.0
Copyright (C) 2023 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

**Comment by TBX3D** [comment]
GOOD NEWS: by changing ``[[clang::fallthrough]];`` to ``[[fallthrough]];`` in ``line 899`` of ``IRToASTVisitor.h``, rellic is successfully built. I will test it and if I find any further issues I will share them here, but it built just fine.

**Comment by TBX3D** [comment]
Well, it seems to be working with the example, so thank you guys for the help! happy to have rellic in my toolkit
![image](https://github.com/user-attachments/assets/e5371628-dc20-4e54-ae90-66d0e3195eb3)
